### PR TITLE
changing how we present gpu attributes

### PR DESCRIPTION
**Pull request by ChristinaLK**

### CODE REVIEW

File: uw-research-computing/gpu-jobs.md [hunk 1]
```diff
@@ -337,11 +337,11 @@ server, including:
 		NVIDIA website</a>.</td>
 	</tr>
 	<tr>
-		<td><code>GPUs_DriverVersion</code></td>
+		<td><code>GPUs_</code>)<code>DriverVersion</code></td>
 		<td><b>Not</b> the version of CUDA on the server, but the maximum CUDA runtime version supported by the GPU drivers on the server. </td>
```
Comment (agitter) [inline uw-research-computing/gpu-jobs.md:341]:
```suggestion
		<td><b>Not</b> the version of CUDA on the server or the NVIDIA driver version, but the maximum CUDA runtime version supported by the NVIDIA driver on the server. </td>
```
This is unrelated to your change, but I find there is still a lot of confusion about this attribute.

**Review by agitter — COMMENTED** [review]
These changes look great to me.  I have one optional suggestion that is slightly related.

It looks like the test case failure is not due to the changes here.

**Comment by agitter** [comment]
As long as we are improving this page, I also recommend we change line 304 to

```
condor_status -constraint 'Gpus > 0' -af Machine TotalGpus GPUs_DeviceName GPUs_Capability
```

`-compact` no longer works well.

**Comment by ChristinaLK** [comment]
@agitter done - look good?

**Review by agitter — APPROVED** [review]

**Pull request merged**